at-quannguyen/W4/main.rb

```diff
@@ -0,0 +1,621 @@
+    puts '1. update name'
+    puts '2. update team leader'
+    print 'Action : '
+    number = begin
+               Integer(gets.chomp)
+             rescue StandardError
+               false
+             end
+    if number == false
+      puts 'input not string !'
+      csleep
+      cexit
+      action_update_team
+    end
+    action_update_team_name if number == 1
+    action_update_team_leader if number == 2
+  end
+
+  def action_update_team_name
+    clear
+    puts '========= Update Name Team ========='
+    print 'name team : '
+    name_team = gets.chomp
+    puts ''
+    print 'name team replace : '
+    name_replace = gets.chomp
+
+    team = @team_controller.select_obj_by_team name_team
+
+    if team.count.zero?
+      puts 'not find team !'
+      csleep
+      cexit
+      action_update_team_name
+    end
+
+    if @team_controller.update_name(name_replace, team.to_a[0]['id'])
+      puts 'update complete !'
+      csleep
+      action_team_manager
+    else
+      puts 'update fail, you try !'
+      csleep
+      cexit
+      action_update_team_name
+    end
+  end
+
+  def action_update_team_leader
+    clear
+    puts '========= Update Leader Team ========='
+    print 'name team : '
+    name_team = gets.chomp
+    puts ''
+    print 'username up leader : '
+    user_up_leader = gets.chomp
+
+    team = @team_controller.select_obj_by_team name_team
+    leader_up = @session_controller.username_by_employee user_up_leader
+
+    if team.count.zero? || leader_up.count.zero?
+      puts 'not find team end not add leader !'
+      csleep
+      cexit
+      action_update_team_leader
+    end
+    @team_controller.update_leader(team, leader_up)
+  end
+
+  def action_notification_manager
+    clear
+    account = @session_controller.current_user
+    query = account.to_a[0]
+    team_id = query['team_id']
+    role = query['role']
+
+    puts '========= Notification Manager ========='
+    puts '1. Send a notification' if role == 2
+    puts '2. Read a notification'
+    puts '6. Delete a notification' if role == 2
+    puts '7. Show list of notification'
+    puts '3. Back'
+    puts '4. Logout'
+    puts '5. Exit program'
+    print 'Action : '
+    number = begin
+               Integer(gets.chomp)
+             rescue StandardError
+               false
+             end
+
+    if number == false
+      puts 'input not string !'
+      csleep
+      cexit
+      action_team_manager
+    end
+
+    if role == 2
+      action_send_notification(team_id) if number == 1
+    end
+
+    action_read_notification(team_id) if number == 2
+    action_list_notifications(team_id) if number == 7
+    action_delete_notification(team_id) if number == 6
+    action_logout_back_exit number
+  end
+
+  def action_send_notification(team_id)
+    clear
+    puts '========= Send Notification ========='
+    print 'Title : '
+    title = gets.chomp
+    puts ''
+    print 'content : '
+    content = gets.chomp
+    current_time = Time.new
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/7deb92b5e1c7275222ca44a289eebcc27cbc38fe/at-quannguyen/W4/main.rb#L413"></span>Rails/TimeZone: Do not use <code>Time.new</code> without zone. Use one of <code>Time.zone.now</code>, <code>Time.current</code>, <code>Time.new.in_time_zone</code>, <code>Time.new.utc</code>, <code>Time.new.getlocal</code>, <code>Time.new.xmlschema</code>, <code>Time.new.iso8601</code>, <code>Time.new.jisx0301</code>, <code>Time.new.rfc3339</code>, <code>Time.new.httpdate</code>, <code>Time.new.to_i</code>, <code>Time.new.to_f</code> instead. (https://github.com/rubocop-hq/rails-style-guide#time, http://danilenko.org/2012/7/6/rails_timezones)</td>
    </tr>
  </tbody>
</table>